ERCS/erc-6860.md

```diff
@@ -293,8 +304,11 @@ contractName    = address
                 / domainName
 chainid         = %x31-39 *DIGIT
 
-pathQuery       = mPathQuery ; path+query for manual mode
-                / aPathQuery ; path+query for auto mode
+pathQuery       = mPathQuery [ evmQuery ] ; path+query for manual mode
+                / aPathQuery [ evmQuery ] ; path+query for auto mode
+
+evmQuery        = *( "&" blockNumberQuery )
```

Comment:
I guess this format cannot cover the case of no query in `mPathQuery` and `aPathQuery`.  E.g., 

`web3://w3url.eth/?_blockNumber=123

Reply:
I tried to fix it, does it meet the standard now?